frontend/src/static/js/v_summary.js

```diff
@@ -172,6 +172,18 @@ window.vSummary = {
                 + `since=${slice.date}'T'00:00:00+08:00&`
                 + `until=${untilDate}'T'23:59:59+08:00`;
     },
+    getRepoLink(repo) {
+      const { REPOS } = window;
+      const urlString = repo.location;
+
+      if (urlString.startsWith('http://github.com/') || urlString.startsWith('https://github.com/')) {
```

Comment:
why not use contains?

Reply:
Because I think that the remote URLs either starts with 'http://github.com/' or 'https://github.com/'. If it is a local repo, the URL will not start with these two.

When we use `startsWith`, it only checks for the keyword at the start of the string.
Whereas `includes` will go through the entire string to check if the keyword is within the
string. So `startsWith` is doing less work which should be faster.

Reply:
Prefer code maintainability over efficiency in cases where the efficiency gain is negligible.

Reply:
ok noted.